.travis.yml

```diff
@@ -14,23 +14,32 @@ compiler: clang
 matrix:
   include:
     - os: linux
-      env: NODE_VERSION=0.12
+      env: NODE_VERSION=4.2.1
     - os: linux
-      env: NODE_VERSION=4
+      env: NODE_VERSION=0.10.35
     - os: osx
-      env: ATOM_SPECS_TASK=core NODE_VERSION=0.12
+      env: NODE_VERSION=4.2.1
     - os: osx
-      env: ATOM_SPECS_TASK=packages NODE_VERSION=0.12
+      env: NODE_VERSION=0.10.35
 
 sudo: false
 
 install:
   - git clone https://github.com/creationix/nvm.git /tmp/.nvm
+  - npm config delete prefix
   - source /tmp/.nvm/nvm.sh
   - nvm install $NODE_VERSION
-  - nvm use $NODE_VERSION
+  - nvm use --delete-prefix $NODE_VERSION
+  - yes | cp apm/package-node-$NODE_VERSION.json apm/package.json
+  - yes | cp build/package-node-$NODE_VERSION.json build/package.json
 
-script: script/cibuild
+script: script/cibuild | ts
```

Comment:
`ts` doesn't seem to exist on Mac.  Is this even needed anyway?  All it does is add a timestamp which I don't think is particularly useful for CI.